assets/src/scripts/contributors.js

```diff
@@ -125,9 +126,9 @@ jQuery( function ( $ ) {
 			 */
 			doCrop: function ( attachment ) {
 				const cropDetails = attachment.get( 'cropDetails' );
-
-				cropDetails.dstWidth  = minPictureSize;
-				cropDetails.dstHeight = minPictureSize;
+				const minSize = parseInt( minPictureSize );
+				cropDetails.dst_width  = minSize;
+				cropDetails.dst_height = minSize;
```

Comment:
@SteelWagstaff those properties will resize the image right after to being cropped and will save it with 400x400 pixels.